helper_test.go

```diff
@@ -613,6 +613,40 @@ func createPolicy(t *testing.T, client *Client, org *Organization) (*Policy, fun
 	}
 }
 
+func createPolicyWithOptions(t *testing.T, client *Client, org *Organization, opts *PolicyCreateOptions) (*Policy, func()) {
+	var orgCleanup func()
+
+	if org == nil {
+		org, orgCleanup = createOrganization(t, client)
+	}
+
+	name := randomString(t)
+	options := PolicyCreateOptions{
```

Comment:
`opts` is a pointer.  This will throw `nil pointer exceptions` if nil is passed in. Do we need to guard against that? Could we just use the struct not the pointer?

Reply:
A struct may be a better and more conventional choice here